ja/controllers/request-response.rst

```diff
@@ -508,16 +508,72 @@ download
         return $response;
     }
 
-ストリーミングリソース
-----------------------
+ヘッダの設定
+------------
+
+.. php:method:: withHeader($header, $value)
+
+ヘッダーの設定は :php:meth:`Cake\\Http\\Response::withHeader()` メソッドで行われます。
+このメソッドは少し違ったパラメータ設定と一緒に呼ばれます。すべての PSR-7 インターフェイスの
```

Comment:
> このメソッドは少し違ったパラメータ設定と一緒に呼ばれます。

This sentence has been already removed in 001536ff7a3b2023c026bf985c92aece9afff064.